crates/bevy_render/src/texture/image_texture_conversion.rs

```diff
@@ -79,34 +79,20 @@ impl Image {
 
                 data = cast_slice(&raw_data).to_owned();
             }
-            DynamicImage::ImageRgb16(image) => {
-                width = image.width();
-                height = image.height();
-                format = TextureFormat::Rgba16Uint;
-
-                let mut local_data =
-                    Vec::with_capacity(width as usize * height as usize * format.pixel_size());
-
-                for pixel in image.into_raw().chunks_exact(3) {
-                    // TODO: use the array_chunks method once stabilised
-                    // https://github.com/rust-lang/rust/issues/74985
-                    let r = pixel[0];
-                    let g = pixel[1];
-                    let b = pixel[2];
-                    let a = u16::max_value();
+            DynamicImage::ImageRgb16(i) => {
```

Comment:
I guess it matches the rest of the file, but I hate that we're using i as a variable name for something that isn't a loop index.

Reply:
agreed, changed